api/v1beta1/helmchart_types.go

```diff
@@ -45,9 +45,15 @@ type HelmChartSpec struct {
 	// +required
 	Interval metav1.Duration `json:"interval"`
 
-	// Alternative values file to use as the default chart values, expected to be a
-	// relative path in the SourceRef. Ignored when omitted.
+	// Alternative list of values files to use as the default chart values,
+	// expected to be a relative path in the SourceRef. Ignored when omitted.
 	// +optional
```

Comment:
I would describe here the merging behavior of the values files, for the helm-controller, we describe tend to describe it as:

> The values are merged in the order given, with the later values overwriting earlier.